Services/GraphApiOperationService.cs

```diff
@@ -0,0 +1,64 @@
+using System;

```

Comment:
Should we just use the Graph SDK ?

Reply:
We can, but the point of this service is to have a resilient HTTP request with the use of client factories in ASP.NET Core. More [here](https://docs.microsoft.com/en-us/dotnet/standard/microservices-architecture/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests). The point is to use best practices, when performing calls to endpoints in a way that feels native to ASP.NET Core developers to "force" developer to use best practices.

Reply:
thanks for the explanation, @bojanv 
this makes sense.